deployment/charts/Chart.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Fuel Core Helm Chart
+name: fuel-core
+version: 1.0.0
```

Since this is likely not the stable version (lots of more changes likely to come) we should downgrade the semver version:

```suggestion
version: 0.1.0
```

---

ok will adjust